source/linux/Overview/Processor_SDK_Linux_create_SD_card.rst

```diff
@@ -56,7 +56,7 @@ Create SD Card using balenaEtcher
 ----------------------------------
 
 1.  Download the default bootable SD card image (WIC file) available on the release page as
-    :file:`tisdk-default-image-<machine>.wic.xz`
+    tisdk-|__IMAGE_TYPE__|-image-<machine>.wic.xz.
```

Comment:
file role was essential in highlighting the SD card image name. Any reasons to remove it?

Reply:
Roles, unfortunately, interfere with substitutions

Reply:
Yeah, as Randolph mentioned the reason the above, roles and substitutions cannot be used together